compiler/rustc_metadata/src/rmeta/def_path_hash_map.rs

```diff
@@ -0,0 +1,91 @@
+use crate::rmeta::DecodeContext;
+use crate::rmeta::EncodeContext;
+use crate::rmeta::MetadataBlob;
+use rustc_data_structures::fingerprint::Fingerprint;
+use rustc_data_structures::owning_ref::OwningRef;
+use rustc_hir::def_path_hash_map::DefPathHashMap as DefPathHashMapInner;
+use rustc_serialize::{opaque, Decodable, Decoder, Encodable, Encoder};
+use rustc_span::def_id::{DefIndex, DefPathHash};
+
+crate struct HashMapConfig;
+
+impl odht::Config for HashMapConfig {
```

Comment:
Can this impl be merged with rustc_hir::def_path_hash_map::Config?

Reply:
Yes.